appveyor.yml

```diff
@@ -10,6 +10,9 @@ environment:
     - PYTHON: "C:\\Python37"
       TOXENV: "py37"
 
+    - PYTHON: "C:\\Python38"
+      TOXENV: "py38"
```

Comment:
This will probably fail, see: https://github.com/PyCQA/pylint/pull/4087#issuecomment-778843277

Reply:
As it's only the column I think we could ignore column number in functional test for windows, wouldn't that be better than not testing at all ?

Reply:
I think this is just an issue with **AppVeyor** not the test itself. It does work as expected with Github Actions, once #4088 is done. For that reason I think it would be best to not include the test for now.

Reply:
It makes sense, I'll do it.